### DISCUSSION TITLE
Request for add support for some few 80286 BIOS clone machines (discussion)

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#3768] [Ideas]
***This is my first feature discussion to support some few other 80286 BIOS clone machines.***

There are 3 BIOS clone files in the zip file. Some 3 BIOS clone files include a readme file before you check at the BIOS files.
List of machines that needs to be added:

1. Mitsubishi 286 clone (came from the Leading Edge MP-2400L board)
    Uses Discrete Logic chipset, maybe [ISA]
    No internal video controller.
    No internal HDD controller.
    Maximum RAM of the Mitsubishi 286 clone is 1MB, maybe extend to 15872KB.
    PS/2 mouse was not supported
    Has 3x 8-bit ISA slots, and 6x 16-bit ISA slots.
    BIOS type: Mitsubishi
    Board information: https://theretroweb.com/motherboards/s/leading-edge-mp-2400l

2. Eurosoft 286 clone (came from the IQS JLI board)
    Uses Headland HT12 chipset, maybe [HT12] or [ISA]
    No internal video controller.
    No internal HDD controller.
    Maximum RAM of the Eurosoft 286 clone is 4MB.
    PS/2 mouse was not supported
    Has 1x 8-bit ISA slots, and 6x 16-bit ISA slots.
    BIOS type: Eurosoft
    Board information: https://theretroweb.com/motherboards/s/iqs-jli

3. Zenith 286 clone (came from the Zenith 85-3261-01 single board)
    Uses IBM PC/AT chipset, maybe [ISA]
    No internal video controller.
    No internal HDD controller.
    Maximum RAM of the Zenith 286 clone is unknown, maybe extend it to 15872KB.
    PS/2 mouse was not supported
    Has 1x 8-bit ISA slots, and 6x 16-bit ISA slots.
    BIOS type: Zenith Data Systems
    Board information: https://theretroweb.com/motherboards/s/zenith-85-3261-01

3 BIOS clone files must be named as:

1. Mitsubishi 286 clone: (mitsubishi286)
2. Eurosoft 286 clone: (eurosoft286)
3. Zenith 286 clone: (zenith286)

BIOS files are here:
[Some 80286 BIOS clone files.zip](https://github.com/86Box/86Box/files/13041990/Some.80286.BIOS.clone.files.zip)
Before using the zip file, don't forget to use the readme file.
(If the BIOS files can be emulated, please let me know.)

**Comment by flama12333** [comment]
Zenith 286 clone are from Z-200 https://www.minuszerodegrees.net/manuals/Zenith%20Data%20Systems/Zenith%20Z-200%20PC%20Series%20-%20Owner's%20Manual.pdf

**Comment by ghost** [comment]
Okay then, I already checked the PDF file. Thanks for sending the owner's manual PDF file.

**Comment by ghost** [comment]
Well, I think I'll close it as outdated, since it was cancelled now due to my (Some unadded ROMs) BIOS ZIP file was at the Discord 86Box Server.
